test/imported/test_indexing.py

```diff
@@ -175,32 +182,57 @@ def validate_setting(x):
     # setting values
     validate_setting(reference)
 
-    # # Tensor with stride != 1
-    # # strided is [1, 3, 5, 7]
-    # reference = consec((10,))
-    # strided = np.array(())
-    # strided.set_(reference.storage(), storage_offset=0,
-    #               size=torch.Size([4]), stride=[2])
-
-    # numpy_testing_assert_equal_helper(strided[[0]], np.array([1]))
-    # numpy_testing_assert_equal_helper(strided[ri([0]), ], np.array([1]))
-    # numpy_testing_assert_equal_helper(strided[ri([3]), ], np.array([7]))
-    # numpy_testing_assert_equal_helper(strided[[1, 2]], np.array([3, 5]))
-    # numpy_testing_assert_equal_helper(strided[ri([1, 2]), ], np.array([3, 5]))
-    # numpy_testing_assert_equal_helper(strided[ri([[2, 1], [0, 3]]), ],
-    #                   np.array([[5, 3], [1, 7]]))
-
-    # # stride is [4, 8]
-    # strided = np.array(())
-    # strided.set_(reference.storage(), storage_offset=4,
-    #               size=torch.Size([2]), stride=[4])
-    # numpy_testing_assert_equal_helper(strided[[0]], np.array([5]))
-    # numpy_testing_assert_equal_helper(strided[ri([0]), ], np.array([5]))
-    # numpy_testing_assert_equal_helper(strided[ri([1]), ], np.array([9]))
-    # numpy_testing_assert_equal_helper(strided[[0, 1]], np.array([5, 9]))
-    # numpy_testing_assert_equal_helper(strided[ri([0, 1]), ], np.array([5, 9]))
-    # numpy_testing_assert_equal_helper(strided[ri([[0, 1], [1, 0]]), ],
-    #                   np.array([[5, 9], [9, 5]]))
+    # Tensor with stride != 1
+    # strided is [1, 3, 5, 7]
+
+    reference = consec((10,))
+
+    # TODO review required
```

Comment:
yeah i can understand this, i think it's fine. but i don't think you want `contiguous()` because that resets stride to 1.

you can add a check to make sure it has the stride you want `strided.lazydata.st.real_strides() == (2,)`